models/intel/inceptionv3-int8-sparse-v2-onnx-0001/description/inceptionv3-int8-sparse-v2-onnx-0001.md

```diff
@@ -0,0 +1,52 @@
+# inceptionv3-int8-sparse-v2-onnx-0001
+
+## Use Case and High-Level Description
+
+This is the Inception v3 model that is designed to perform image classification. 
```

Comment:
If original model exists in OMZ/public, please add link to it.

Reply:
@eizamaliev link to original model not required there